charger/kathrein.go

```diff
@@ -373,13 +368,12 @@ func (wb *Kathrein) Phases1p3p(phases int) error {
 	}
 
 	// Disable and re-enable charging to apply the new phase setting
+	if err := wb.Enable(false); err != nil {
```

Comment:
Why aren't we checking for enabled here? This is a nop if we're already disabled?